dylint/src/lib.rs

```diff
@@ -503,7 +516,7 @@ fn check_or_fix(opts: &Dylint, resolved: &ToolchainMap) -> Result<()> {
         // https://github.com/rust-lang/rust-clippy/commit/1a206fc4abae0b57a3f393481367cf3efca23586
         // But I am going to continue to set CLIPPY_DISABLE_DOCS_LINKS because it doesn't seem to
         // hurt and it provides a small amount of backward compatibility.
-        let result = command
+        let mut result = command
```

Comment:
```suggestion
        command
```

And change the `result` to `command` between here and line 554 below.